### PR TITLE
🐛[bento-sidebar] Remove accidentally rendered panel from sidebar. 

### LINKED ISSUE #36929: Visual glitch in bento-sidebar
### Description

There's a grey vertical bar visiable when the bento-sidebar is placed to the right:

image

### Reproduction Steps

Click on "Open sidebar": https://jsbin.com/ralojodapu/edit?html,output

### Relevant Logs

_No response_

### Browser(s) Affected

Chrome 95.0.4638.69 

### OS(s) Affected

_No response_

### Device(s) Affected

_No response_

### AMP Version Affected

Current beta channel version.

### PR DESCRIPTION
**Pull request by rebeccanthomas**
Currently when a sidebar is set to display on the right hand side of the window, a solid opaque bar is accidentally rendered on the left side of the window. This is due to a CSS rule which is being applied incorrectly to the sidebar 
'wrapper' component. 
Fixes #36929.

Before:
image

After: 
image

### CODE REVIEW

**Review by dmanek — APPROVED** [review]

File: extensions/amp-sidebar/1.0/amp-sidebar.css [hunk 1]
```diff
@@ -30,18 +22,25 @@ amp-sidebar::part(sidebar) {
   background-color: inherit;
   height: inherit;
   top: inherit;
-  max-height: inherit;
-  max-width: inherit;
-  min-width: inherit;
```
Comment (dmanek) [inline extensions/amp-sidebar/1.0/amp-sidebar.css:35]: Just confirming that these styles are also not required for `amp-sidebar::part(c)` and `amp-sidebar::part(wrapper)`?

Comment (rebeccanthomas) [reply extensions/amp-sidebar/1.0/amp-sidebar.css:35]: Thats correct. The sidebar still renders as expected without those styles for `part(c)` and `part(wrapper)`.

File: extensions/amp-sidebar/1.0/amp-sidebar.css [hunk 2]
```diff
@@ -10,6 +10,8 @@ amp-sidebar {
   background-color: #efefef;
   height: 100vh;
   top: 0;
+  outline: none;
```
Comment (dmanek) [inline extensions/amp-sidebar/1.0/amp-sidebar.css:13]: FYI setting outline: none breaks a11y if this element needs to receive focus (https://developer.mozilla.org/en-US/docs/Web/CSS/outline#accessibility_concerns).

File: extensions/amp-sidebar/1.0/amp-sidebar.css [hunk 3]
```diff
@@ -10,6 +10,8 @@ amp-sidebar {
...
+  z-index: 2147483647;
```
Comment (dmanek) [inline extensions/amp-sidebar/1.0/amp-sidebar.css:14]: I did not find any other instance in our code base where we set z-index this high. Not sure if we want to start some kind of z-index arms race 🙂

File: extensions/amp-sidebar/1.0/amp-sidebar.css (hunk 2 above)
Comment (rebeccanthomas) [reply extensions/amp-sidebar/1.0/amp-sidebar.css:13]: This value was set when the component styling was done initially [here](https://github.com/ampproject/amphtml/commit/7311ed960f9aa71e5160095e3cb64db75fa037f6#diff-ff22105e57037561cfba33a82c88dcf4243939ddcc5318ed895143510eafe913R30-R31)

However, In order to make the sidebar receive focus maybe we should change this in a follow up pr?

File: extensions/amp-sidebar/1.0/amp-sidebar.css (hunk 3 above)
Comment (rebeccanthomas) [reply extensions/amp-sidebar/1.0/amp-sidebar.css:14]: This value was set when the component styling was done initially [here](https://github.com/ampproject/amphtml/commit/7311ed960f9aa71e5160095e3cb64db75fa037f6#diff-ff22105e57037561cfba33a82c88dcf4243939ddcc5318ed895143510eafe913R30-R31)

I believe this is to make sure that any sidebar displays on top of anything on the page.

**Pull request merged**